From 5a17d3abd84c82bbc47e16a552ef6325a88af36c Mon Sep 17 00:00:00 2001
From: Graham Leggett <minfrin@apache.org>
Date: Tue, 18 Sep 2018 21:24:06 +0000
Subject: [PATCH 168/504] mod_proxy_hcheck: Fix issues with TCP health checks.
 PR 61499 trunk patch: http://svn.apache.org/r1840582 2.4.x patch: svn merge
 -c 1840582 ^/httpd/httpd/trunk . +1: jim, ylavic, minfrin

git-svn-id: https://svn.apache.org/repos/asf/httpd/httpd/branches/2.4.x@1841263 13f79535-47bb-0310-9956-ffa450edef68
---
 CHANGES                          | 3 +++
 STATUS                           | 4 ----
 modules/proxy/mod_proxy_hcheck.c | 1 -
 3 files changed, 3 insertions(+), 5 deletions(-)

diff --git a/modules/proxy/mod_proxy_hcheck.c b/modules/proxy/mod_proxy_hcheck.c
index 0ee6845265..bf557b71e9 100644
--- a/modules/proxy/mod_proxy_hcheck.c
+++ b/modules/proxy/mod_proxy_hcheck.c
@@ -612,7 +612,6 @@ static apr_status_t hc_check_tcp(baton_t *baton)
 
     status = hc_get_backend("HCTCP", &backend, hc, ctx, baton->ptemp);
     if (status == OK) {
-        backend->addr = hc->cp->addr;
         status = ap_proxy_connect_backend("HCTCP", backend, hc, ctx->s);
         /* does an unconditional ap_proxy_is_socket_connected() */
     }
-- 
2.19.1

